vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-# Business Objective
+---
+sidebarDepth: 2
+---
 
-The Entando Process Driven Applications plugin is built to provide a
-rich and full featured user experience to allow for the management and
-completion of business processes and automation. The UX is delivered a
-general purpose UX layer built using micro frontends that can be
-utilized for any business process or task engine. Customers drive the UX
-layer by utilizing a pre-developed Entando integration or by
-implementing a set of interfaces on the server side. The backend is a
-Spring Boot microservice that provides a pluggable interface allowing
-for the injection of any underlying process or automation toolkit. The
-interfaces and steps for creating a new PDA backend implementation are
-provided below. This document provides an overview of the Entando
-Process Driven Apps (PDA) plugin. The initial section covers the
-business functionality and user facing micro frontends. The second
-section covers the architecture and the points of extension for the PDA
-plugin. The third provides a detailed technical overview for developers
-to add new functionality to a PDA implementation.
+# Entando Process Driven Applications Plugin
 
-# Terminology
+## Introduction
 
-## Micro Frontend
+The Entando Process Driven Applications (PDA) plugin is engineered to provide a rich and full-featured user experience while facilitating the management and completion of business processes and automation.  The solution template includes: 
```

Comment:
Maybe "This solution template" rather than "The solution template"

Reply:
done